src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -207,8 +208,20 @@ class GcsServer {
   /// Stats handler and service.
   std::unique_ptr<rpc::StatsHandler> stats_handler_;
   std::unique_ptr<rpc::StatsGrpcService> stats_service_;
-  // Synchronization service for ray.
-  std::unique_ptr<gcs_syncer::RaySyncer> ray_syncer_;
+
+  /// Synchronization service for ray.
+  /// TODO(iycheng): Deprecate this gcs_ray_syncer_ one once we roll out
+  /// to ray_syncer_.
+  std::unique_ptr<gcs_syncer::RaySyncer> gcs_ray_syncer_;
```

Comment:
should `gcs_syncer::RaySyncer` and `syncer::RaySyncer` shared the same interface and we only have one instance of it in gcs_server?

Reply:
I don't think we should do that. It's not necessary given that the old one will be deleted after rolling. Besides, I feel if we do it this way, it's going to make the code more complicated since the API in these two classes are different and also how it's used inside the class.